src/components/UserApplyButton/UserApplyButton.tsx

```diff
@@ -30,7 +31,11 @@ const UserApplyButton = ({ eventId, eventDetail, applyModalOpen }: UserApplyButt
           {applicants}/{capacity}
         </ApplicantButton>
       )}
-      <ApplyButton capacity={!!capacity} onClick={() => applyModalOpen()}>
+      <ApplyButton
+        capacity={!!capacity}
```

Comment:
p5;
```suggestion
        capacity={Boolean(capacity)}
```
이렇게 선언적인 코드는 어떨까요?

Reply:
오호~ 이런 방법도 있었군요! 감사합니다:)